AirportTicketNookingTest/BookingShould.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Airport_Ticket_Booking;
+using AutoFixture;
+using FluentAssertions;
+using Xunit;
+using static Airport_Ticket_Booking.Common;
+
+namespace AirportTicketNookingTest
+{
+    public class BookingShould
+    {
+        private readonly Fixture _fixture;
+        private readonly FlightsManager _flightManager;
+
+        public BookingShould()
+        {
+            _fixture = new Fixture();
+            _flightManager = FlightsManager.Instance();
+
+            _flightManager.ReadFromCSV(@"Flights.csv", true);
```

Comment:
you're reading a file here that makes your test depend on an external state. This is where you can make use of mock.